libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/Edge.kt

```diff
@@ -0,0 +1,155 @@
+package com.mapbox.navigation.base.trip.model
+
+import com.mapbox.geojson.LineString
+
+/**
+ * Basic Edge class
+ */
+data class Edge(
```

Comment:
Same as above.

Reply:
@mskurydin @SiarheiFedartsou could you help here? Are we envisioning extending this / adding more data as part of `Edge` in the future?

Reply:
Can easily happen.